Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.py

```diff
@@ -1776,6 +1779,28 @@ def set_password_not_expire(default_base_dn):
     else:
         raise DemistoException(f"Unable to fetch attribute 'userAccountControl' for user {sam_account_name}.")
 
+def test_credentials():
+    args = demisto.args()
+    SERVER_IP = demisto.params().get('server_ip')
```

Comment:
this commit 9e22e29c593faeeb8b7909fee7d94df8a53fb883 contains the change to pass SERVER_IP as an argument to the test_credentials functions since it's already defined in main